main.js

```diff
@@ -36,11 +36,15 @@ function createMenu() {
   // The clicks here take a function so that additional parameters such as
   // a pointer to the menu item can be passed.
   let template = [
-    {
-      label: 'Check for updates...',
-      click: updater.checkForUpdates
-    },
-    {type: 'separator'},
+    // 2018-09-12 mp: Disabling check for updates  until we update the update process to use
+    // omnitruck so that users don't get message around missing app-update.yml
```

Comment:
I think I've set a new record for the number of times 'update' is used in a sentence.